process/charter.html

```diff
@@ -485,7 +485,9 @@ <h3>5.2 <a id="review-eval" name="review-eval">Evaluating the
 
 <h3 id="managing-changes">5.3 Managing Changes to Charters</h3>
 
-<p>Very minor changes may be made in place to a charter. If there are only very minor changes to a charter resulting from the review, the (future) Director's decision includes the original charter URI.</p>
+<p>Very minor changes may be made in place to a charter. If there are only very minor changes to a charter resulting from the review, the (future) Director's decision includes the original charter URI and an explanation of the changes and their rationale.</p>
+<p>If substantive changes are proposed in response to charter review, Team must send the charter with proposed changes, a diff, and an explanation of the changes and their rationale to all who reviewed the charter, copying the member-charters-review@w3.org member-archived mailing list, accompanied by a request for response (in no less than one week) if reviewers have concerns or if the changes would alter their reviews. If the work continues or derives from an existing Group or community effort, Team should also send the diff and a public rationale to that Group or community. These communications should note that the Director has not yet approved the charter.</p>
+<p>If proposed changes are substantial, or if anyone expresses concern in response to the re-review, Team should formally open a new AC charter review.</p>
```

Comment:
This seems to always require a new AC review if there are substantive changes, which contradicts the paragraph above. If none of the reviewers objects to the change after a week review, why do a new AC review?

Reply:
I was differentiating between "substantial" (major) and substantive (could be major or minor). For major changes, yes, I was saying they should by default go to new AC review. 

Should I remove that clause?

Reply:
```suggestion
<p>If anyone expresses new concern in response to the re-review, Team may attempt to resolve the concern (with re-review), formally open a new AC charter review, or the Director may treat it as an objection and overrule it.</p>
```